specification/hybridconnectivity/resource-manager/Microsoft.HybridConnectivity/stable/2023-03-15/hybridconnectivity.json

```diff
@@ -0,0 +1,1093 @@
+        ],
+        "x-ms-examples": {
+          "ServiceconfigurationsPatchSSH": {
+            "$ref": "./examples/ServiceconfigurationsPatchSSH.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceUriParameter"
+          },
+          {
+            "$ref": "#/parameters/EndpointNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ServiceConfigurationNameParameter"
+          },
+          {
+            "name": "ServiceConfigurationResource",
+            "description": "Service details",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ServiceConfigurationResourcePatch"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "ServiceConfiguration service.",
+            "schema": {
+              "$ref": "#/definitions/ServiceConfigurationResource"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "Serviceconfigurations"
+        ],
+        "description": "Deletes the service details to the target resource.",
+        "operationId": "Serviceconfigurations_Delete",
+        "produces": [
+          "application/json"
+        ],
+        "x-ms-examples": {
+          "ServiceconfigurationsDeleteSSH": {
+            "$ref": "./examples/ServiceconfigurationsDeleteSSH.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceUriParameter"
+          },
+          {
+            "$ref": "#/parameters/EndpointNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ServiceConfigurationNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The endpoint is deleted successfully."
+          },
+          "204": {
+            "description": "The endpoint has been deleted already."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/{resourceUri}/providers/Microsoft.HybridConnectivity/endpoints/{endpointName}/serviceconfigurations": {
```

Comment:
>serviceconfigurations [](http://example.com/codeflow?start=84&length=21)

camelCase serviceConfigurations. 
[Also, personally, I like to put the list API above the GET api, so you have your two GET apis right next to each other]

Reply:
Changed to camelCase. Also moved it above GET API.